xarray/core/dataset.py

```diff
@@ -5146,5 +5149,106 @@ def filter_by_attrs(self, **kwargs):
                 selection.append(var_name)
         return self[selection]
 
+    def unify_chunks(self):
+        """ Unifies chunksize along all chunked dimensions of this Dataset.
```

Comment:
This could greatly benefit from being moved to top-level with thin wrappers in DataArray and Dataset, so that one can unify chunks across multiple DataArray's and Dataset's.

Reply:
Yeah that would be good. Let's leave that to a future PR.